docker/default.env

```diff
@@ -0,0 +1,38 @@
+# Default configuration for Docker containers
+# To modify, copy these over to a ".env" file
+# and values here will be overridden.
+#
+# Values passed via command-line arguments take precedence over .env files.
+#
+# Note that there is no special handling of quotation marks.
+# This means that they are part of the value.
+#
+# Note that these variables are not available at docker-compose.yml
+# Variables show up defined only inside containers.
+
+# WordPress
+WP_DOMAIN=localhost
+WP_ADMIN_USER=wordpress
+WP_ADMIN_EMAIL=wordpress@example.com
+WP_ADMIN_PASSWORD=wordpress
+WP_TITLE=HelloWord
+
+# Database
+MYSQL_HOST=db:3306
+MYSQL_DATABASE=wordpress
+MYSQL_USER=wordpress
+MYSQL_PASSWORD=wordpress
+MYSQL_ROOT_PASSWORD=wordpress
+
+# SFTP container users (user:pass:UID)
+#
+# Set UID/GID manually for your users if you want them to make changes to
+# your mounted volumes with permissions matching your host filesystem.
+#
+# Define multiple users separated by space
+#
+# Read more: https://github.com/atmoz/sftp
+SFTP_USERS=wordpress:wordpress:1001
+
+# Xdebug
+PHP_IDE_CONFIG=serverName=Test
```

Comment:
Is this something anyone would ever want to modify? If not, I'll move it back to `docker-compose.yml`

Reply:
I think PhpStorm will choose the server configuration according to this. Haven't used this environment variable myself by modifying but looks useful